src/datapane/client/api/user.py

```diff
@@ -121,17 +121,15 @@ def signup():
     # login and re-init against the signed-up server
     login(token, server=config.server)
 
-    display_msg("\nRunning `./world.py` and uploading the Datapane report to your newly created account.\n")
-
-    # run the world script
-    _run_script("world.py")
-
-    # success_msg("Report uploaded, opening in your browser")
     display_msg(
         "\nLearn more about uploading and sharing reports {learn_url:l}.",
         learn_url="https://docs.datapane.com/reports/publishing-and-sharing",
     )
 
+    display_msg(
+        "\nWe’d also love to invite you to our community spaces for a chat (https://chat.datapane.com), forum discussion (https://forum.datapane.com), and open source collaboration (https://github.com/datapane/datapane)."
```

Comment:
you can use the `l` cmd specific to create a markdown link - or add new cmd specifier to handle a bit differently if needed rather than links in brackets

Reply:
Done!